### ISSUE TITLE
List of unusual files in the repo

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fxa#12830]
## Description

Probably of zero interest to anybody that isn't me, but I was trying to see if we still had a stray .jsx file in the repo and came up with this horrific combination of `git ls-files` and chained `grep` commands.

One interesting thing is that we seem to be using .hbs/.handlebars, .ejs, and .mustache templating engines throughout the codebase.

### Steps to reproduce

```sh
git ls-files packages/ | grep -Ev "\.(js|json|tsx?|md|ejs|hbs|handlebars|mustache|ftl|s?css|svg|png|gif|jpg|docker|sh|html|nsprc|eslintrc|eslintignore|prettierignore|gitignore|ico|txt|mjml|nvmrc|htmllintrc|gitattributes|eot|ttf|woff2?|sql|gql|netset|mmdb|crt|key|graffle|lua|ini|json|json-dist|toml|ya?ml)$" | grep -Ev "/(ansible|teamcity|123done.sketch)/" | grep -Ev "(AUTHORS|LICENSE|Makefile|Procfile|tap-parallel-not-ok)$"

packages/browserid-verifier/loadtest/loadtest.py
packages/db-migrations/bin/patcher.mjs
packages/fxa-auth-server/test/load/loadtests.py
packages/fxa-dev-launcher/bin/fxa-dev-launcher.mjs
packages/fxa-dev-launcher/profile.mjs
packages/fxa-geodb/scripts/mmdb/create-mmdb.pl
packages/fxa-geodb/scripts/mmdb/read-mmdb.pl
packages/fxa-settings/.license.header
```

### Expected result

🤷 

### Actual result

TIL we have a couple Python files (which I think are orphaned/broken load tests), and [maxmind-related] Perl files in the repo (as well as fancy .mjs files).

### Environment

n/a

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-5065)

**Comment by LZoog** [comment]
`packages/fxa-settings/.license.header` is there to serve as a base for our l10n concatenation. It is sorta odd though.

I know we've had to convert some JS files explicitly to `.mjs` to tell something or another that it was an ES6 module. I can't remember why but I know at least one of those fixed some broken thing.

> ...  .hbs/.handlebars, .ejs, and .mustache templating engines throughout the codebase.

I think anything hbs/mustache is in the content-server right? 👀

I can't speak to why we've got some of that Python stuff well, in Python (seems many years old), but I didn't even know we had anything in Perl - maybe we can take a look at that when we upgrade maxmind in #12829

**Comment by pdehaan** [comment]
> I think anything hbs/mustache is in the content-server right? 👀

TLDR:
- fortress/*.ejs (6)
- fxa-auth-server/*.handlebars (1)
- fxa-content-server/*.mustache (86)
- fxa-support-panel/*.hbs (1)

```sh
git ls-files packages/**/*.{ejs,hbs,handlebars,mustache}

packages/fortress/views/choice.ejs
packages/fortress/views/default.ejs
packages/fortress/views/download.ejs
packages/fortress/views/footer.ejs
packages/fortress/views/header.ejs
packages/fortress/views/index.ejs

packages/fxa-auth-server/scripts/api-docs.handlebars

packages/fxa-content-server/app/scripts/templates/400.mustache
packages/fxa-content-server/app/scripts/templates/500.mustache
packages/fxa-content-server/app/scripts/templates/account_recovery_confirm_key.mustache
packages/fxa-content-server/app/scripts/templates/cannot_create_account.mustache
packages/fxa-content-server/app/scripts/templates/choose_what_to_sync.mustache
packages/fxa-content-server/app/scripts/templates/clear_storage.mustache
packages/fxa-content-server/app/scripts/templates/complete_reset_password.mustache
packages/fxa-content-server/app/scripts/templates/complete_sign_up.mustache
packages/fxa-content-server/app/scripts/templates/confirm.mustache
packages/fxa-content-server/app/scripts/templates/confirm_reset_password.mustache
packages/fxa-content-server/app/scripts/templates/confirm_signup_code.mustache
packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache
packages/fxa-content-server/app/scripts/templates/cookies_disabled.mustache
packages/fxa-content-server/app/scripts/templates/force_auth.mustache
packages/fxa-content-server/app/scripts/templates/index.mustache
packages/fxa-content-server/app/scripts/templates/inline_recovery_setup.mustache
packages/fxa-content-server/app/scripts/templates/inline_totp_setup.mustache
packages/fxa-content-server/app/scripts/templates/legal.mustache
packages/fxa-content-server/app/scripts/templates/loading.mustache
packages/fxa-content-server/app/scripts/templates/marketing_snippet.mustache
packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache
packages/fxa-content-server/app/scripts/templates/pair/auth_complete.mustache
packages/fxa-content-server/app/scripts/templates/pair/auth_wait_for_supp.mustache
packages/fxa-content-server/app/scripts/templates/pair/failure.mustache
packages/fxa-content-server/app/scripts/templates/pair/index.mustache
packages/fxa-content-server/app/scripts/templates/pair/success.mustache
packages/fxa-content-server/app/scripts/templates/pair/supp.mustache
packages/fxa-content-server/app/scripts/templates/pair/supp_allow.mustache
packages/fxa-content-server/app/scripts/templates/pair/supp_wait_for_auth.mustache
packages/fxa-content-server/app/scripts/templates/pair/unsupported.mustache
packages/fxa-content-server/app/scripts/templates/partial/account-suggestion.mustache
packages/fxa-content-server/app/scripts/templates/partial/coppa-age-input.mustache
packages/fxa-content-server/app/scripts/templates/partial/device-being-paired.mustache
packages/fxa-content-server/app/scripts/templates/partial/email-autocomplete-domains.mustache
packages/fxa-content-server/app/scripts/templates/partial/firefox-family-services.mustache
packages/fxa-content-server/app/scripts/templates/partial/password-strength-balloon.mustache
packages/fxa-content-server/app/scripts/templates/partial/permission.mustache
packages/fxa-content-server/app/scripts/templates/partial/settings-header.mustache
packages/fxa-content-server/app/scripts/templates/partial/show-password.mustache
packages/fxa-content-server/app/scripts/templates/partial/support-form-error.mustache
packages/fxa-content-server/app/scripts/templates/partial/support-form-success.mustache
packages/fxa-content-server/app/scripts/templates/partial/sync-suggestion.mustache
packages/fxa-content-server/app/scripts/templates/partial/third-party-auth.mustache
packages/fxa-content-server/app/scripts/templates/partial/user-card.mustache
packages/fxa-content-server/app/scripts/templates/permissions.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/account_recovery/add_recovery_key.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/account_recovery/confirm_password.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/account_recovery/confirm_recovery_key.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/account_recovery/save_recovery_key.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/cad_qr/connected.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/cad_qr/get_started.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/cad_qr/ready_to_scan.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/cad_qr/scan_code.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/finish_account_setup/set_password.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/newsletters/add_newsletters.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/password/force_password_change.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/secondary_email/add_secondary_email.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/secondary_email/confirm_secondary_email.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/third_party_auth/callback.mustache
packages/fxa-content-server/app/scripts/templates/post_verify/verified.mustache
packages/fxa-content-server/app/scripts/templates/pp.mustache
packages/fxa-content-server/app/scripts/templates/push/completed.mustache
packages/fxa-content-server/app/scripts/templates/push/confirm_login.mustache
packages/fxa-content-server/app/scripts/templates/push/send_login.mustache
packages/fxa-content-server/app/scripts/templates/ready.mustache
packages/fxa-content-server/app/scripts/templates/redirect_loading.mustache
packages/fxa-content-server/app/scripts/templates/report_sign_in.mustache
packages/fxa-content-server/app/scripts/templates/reset_password.mustache
packages/fxa-content-server/app/scripts/templates/security_events.mustache
packages/fxa-content-server/app/scripts/templates/settings/account_recovery/confirm_password.mustache
packages/fxa-content-server/app/scripts/templates/settings/account_recovery/recovery_key_print_template.mustache
packages/fxa-content-server/app/scripts/templates/settings/recovery_codes_print.mustache
packages/fxa-content-server/app/scripts/templates/sign_in_bounced.mustache
packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache
packages/fxa-content-server/app/scripts/templates/sign_in_recovery_code.mustache
packages/fxa-content-server/app/scripts/templates/sign_in_reported.mustache
packages/fxa-content-server/app/scripts/templates/sign_in_token_code.mustache
packages/fxa-content-server/app/scripts/templates/sign_in_totp_code.mustache
packages/fxa-content-server/app/scripts/templates/sign_in_unblock.mustache
packages/fxa-content-server/app/scripts/templates/sign_up_password.mustache
packages/fxa-content-server/app/scripts/templates/sub_panels.mustache
packages/fxa-content-server/app/scripts/templates/support.mustache
packages/fxa-content-server/app/scripts/templates/test_template.mustache
packages/fxa-content-server/app/scripts/templates/tos.mustache
packages/fxa-content-server/app/scripts/templates/why_connect_another_device.mustache
packages/fxa-content-server/app/scripts/templates/would-you-like-to-sync.mustache

packages/fxa-support-panel/views/index.hbs
```

**Comment by LZoog** [comment]
Questions I have (probably not for you Peter, just thinking outloud):
* What are using fortress for at this point? I always access 123Done to get to various flows with fake products.
* Do we need `api-docs.handlebars` any longer now that we've got Swagger?
* Would be low priority, but should we file an issue to convert support-panel to EJS? 

If we don't use fortress for anything and can remove `api-docs.handlebars` and convert support-panel over to EJS, whenever content-server is eventually converted over to React then our UIs will all be consistent between React or EJS.

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

We want to remove fortress: https://mozilla-hub.atlassian.net/browse/FXA-2983 ( https://mozilla-hub.atlassian.net/browse/FXA-2983|smart-link ) 

Let’s get rid of the api-docs stuff.  It’s all super old and unused.

Convert to EJS sounds good

**Comment by data-sync-user** [comment]
➤ Lauren Zugai commented:

I just filed https://mozilla-hub.atlassian.net/browse/FXA-5155 ( https://mozilla-hub.atlassian.net/browse/FXA-5155|smart-link ) and https://mozilla-hub.atlassian.net/browse/FXA-5154 ( https://mozilla-hub.atlassian.net/browse/FXA-5154|smart-link ), and with https://mozilla-hub.atlassian.net/browse/FXA-2983 ( https://mozilla-hub.atlassian.net/browse/FXA-2983|smart-link ) existing too, I think we’re good with these files. Thanks Peter deHaan, I’ll close this ticket now. 👍
